### PR TITLE
fix help text onfocus and onblur

### PR DESCRIPTION
**Pull request by jasonephraim**

### CODE REVIEW

File: rigex-front-end/src/components/auth/signUpForm.js [hunk 1]
```diff
@@ -25,9 +25,12 @@ const SignUpForm = () => {
     email: '',
     password: '',
     showPassword: false,
+    showHelpText: false,
   });
   const [errorMessage, setErrorMessage] = useState('');
-  const [helpMessage, setHelpMessage] = useState('');
```
Comment (aaronpaulroberts) [inline rigex-front-end/src/components/auth/signUpForm.js:30]: What was the need for this change?

Comment (jasonephraim) [reply rigex-front-end/src/components/auth/signUpForm.js:30]: The field's onFocus wouldn't revert when the field was no longer active. I had to add another prop to the field for onBlur and this state needed to toggle depending on if the field was focused or not

**Review by mmevans — APPROVED** [review]

File: rigex-front-end/src/components/auth/signUpForm.js (hunk 1 above)
Comment (gr8white) [reply rigex-front-end/src/components/auth/signUpForm.js:30]: Shouldn't this message only show up when there's a problem with the user input?

Comment (jasonephraim) [reply rigex-front-end/src/components/auth/signUpForm.js:30]: the designs show it appears when the field is focused. Barrette included the comment it should disappear when it is not in focus.

**Review by BarretteBanner — APPROVED** [review]

**Pull request merged**